Programm started at: 2025-08-25 02:01:39

Running in version 1.2[2025-08-25 22:01:07]
Error 1 at reading or storing sensor data.
	'I2C read failed'

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 191, in read_all
    if self.data_available():
       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 172, in data_available
    value = self.pi.i2c_read_byte_data(self.device, CSS811_STATUS)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 2874, in i2c_read_byte_data
    return _u2i(_pigpio_command(self.sl, _PI_CMD_I2CRB, handle, reg))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 1011, in _u2i
    raise error(error_text(v))
pigpio.error: 'I2C read failed'


[2025-08-25 23:32:14]
Git repositories synchronized
	

	NoneType: None


[2025-08-25 23:32:14]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-08-26 02:02:42

Running in version 1.2

[2025-08-26 23:32:16]
Git repositories synchronized
	

	NoneType: None


[2025-08-26 23:32:16]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-08-27 02:02:36

Running in version 1.2

[2025-08-27 23:32:10]
Git repositories synchronized
	

	NoneType: None


[2025-08-27 23:32:10]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-08-28 02:01:47

Running in version 1.2

[2025-08-28 23:32:21]
Git repositories synchronized
	

	NoneType: None


[2025-08-28 23:32:21]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-08-29 02:02:49

Running in version 1.2

[2025-08-29 23:28:19]
Error 1 at reading or storing sensor data.
	'I2C read failed'

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 191, in read_all
    if self.data_available():
       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 172, in data_available
    value = self.pi.i2c_read_byte_data(self.device, CSS811_STATUS)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 2874, in i2c_read_byte_data
    return _u2i(_pigpio_command(self.sl, _PI_CMD_I2CRB, handle, reg))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 1011, in _u2i
    raise error(error_text(v))
pigpio.error: 'I2C read failed'


